readme.md

```diff
@@ -891,6 +891,7 @@
 - [Whisper](https://github.com/sindresorhus/awesome-whisper#readme) - Open-source AI-powered speech recognition system developed by OpenAI.
 - [Steam Deck](https://github.com/airscripts/awesome-steam-deck#readme) - A handheld gaming computer developed by Valve.
 - [HPC](https://github.com/dstdev/awesome-hpc#readme) - High Performance Computing.
+- [Hybris](https://github.com/eminyagiz42/awesome-hybris#readme) - An e-commerce platform built with Java,Spring MVC and Angular.
```

Comment:
Since Hybris was acquired by SAP and rebranded to SAP Commerce, I suggest changing the repository name and title to it. 


```suggestion
- [SAP Commerce](https://github.com/eminyagiz42/awesome-sap-commerce#readme) - An e-commerce platform built with Java, Spring MVC, and Angular.
```

Reply:
Hello @virtualroot ,  Thanks for the suggestions, but i rather keep it as hybris. This term still is actively being used among developers. Changing the title in the future might be good idea.